app/services/yoksis/v1/ozgecmis.rb

```diff
@@ -16,6 +16,7 @@ def initialize
           )
         end
 
+        # TODO: Will refactor.
```

Comment:
Bu not aşağıdaki `message` bloğu için. Tüm servisleri ekledikten sonra buna bir çeki düzen vereceğim. Şimdilik diğer servislerde nasıl çalıştığına bakmadığım için bu şekilde bırakıyorum.